File: pkg/java/src/main/java/dev/openfga/language/errors/SimpleError.java
package dev.openfga.language.errors;

import com.fasterxml.jackson.annotation.JsonProperty;

public abstract class SimpleError {

    @JsonProperty("msg")
    private String message;

    public SimpleError() {}

    public SimpleError(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String toString() {
        return message;
    }
}

File: pkg/csharp/src/OpenFga.Language/Errors/SimpleError.cs
using SharpYaml.Serialization;
using System.Text.Json.Serialization;

namespace OpenFga.Language.Errors;

public abstract class SimpleError {
    [JsonPropertyName("msg")]
    [YamlMember("msg")]
    public string Message { get; set; } = string.Empty;

    protected SimpleError() { }

    protected SimpleError(string message) {
        Message = message;
    }

    public override string ToString() {
        return Message;
    }
}